src/components/Header/Header.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import CustomNavLink from '../CustomNavLink'
+import { connect } from 'react-redux'
+import { logoutUser } from '../../actions/AuthActions'
+import {
+  HeaderImage,
+  NavListItem,
+  LogoutText,
+  LogoutDiv,
+  TopNav
+} from './styled'
+
+const Header = ({ logoutUser }) => {
+  let toLinks = ['/fbconnect', '/igconnect', '/twconnect']
```

Comment:
I'd rather create an array of objects here, and each object would contain link and the linkText respectively.